.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Lint
+on: [push, pull_request]
```

Comment:
Shouldn't this be only on pull_request?

Reply:
@stianst that's up to you and @slaskawi. The advantage of having this on push, is the fact that you can double check it if you push new tags. But honestly, that's your choice.